text/0039-json-schemas/service.json

```diff
@@ -0,0 +1,255 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://obsproject.com/schemas/service.json",
+    "title": "OBS Service Object Schema",
+    "type": "object",
+    "properties": {
+        "more_info_link": {
+            "type": "string",
+            "description": "Link that provides additional info about the service, presented in the UI as a button next to the services dropdown.",
+            "format": "uri",
+            "$ref": "#/$defs/httpPattern",
+            "minLength": 1
+        },
+        "stream_key_link": {
+            "type": "string",
+            "description": "Link where a logged-in user can find the 'stream key', presented as a button alongside the stream key field.",
+            "format": "uri",
+            "$ref": "#/$defs/httpPattern",
+            "minLength": 1
+        },
+        "servers": {
+            "type": "array",
+            "description": "Array of server objects",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "name": {
+                        "type": "string",
+                        "description": "Name of the server (e.g. location, primary/backup), displayed in the Server dropdown.",
+                        "minLength": 1
+                    },
+                    "protocol": {
```

Comment:
Just a thought that different protocols might want different encoder settings

Reply:
I try to avoid making a mess, and protocol-agnostic per codec settings is the simplest and the less messy.

Reply:
If the service wants to do more than what `obs-services` allows, it can create a third-party service plugin.